src/block/Block.php

```diff
@@ -482,6 +495,13 @@ public function isAffectedBySilkTouch() : bool{
 		return false;
 	}
 
+	/**
+	 * Returns whether Fortune enchanted tools will cause this block to drop more items.
+	 */
+	public function isAffectedByFortune() : bool{
```

Comment:
I get the impression this method is probably redundant. If a block isn't affected by fortune, it won't override getFortuneDrops(), and the block's regular drops will be given anyway.